### ISSUE TITLE
Is it possible to poll a poller

### ISSUE THREAD

**Issue by DRAKUN** [centreon/centreon#5834]
Hi every body is it possible to have this architecture with Centreon
[https://drive.google.com/open?id=0B8XHICQ3quvdMDFXb2gxMkZvMm8](url)

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 
Latest
**Centreon Engine version**: 
Latest
**Centreon Broker version**: 
Latest
**OS**: 
Vm
**Additional environment details (AWS, VirtualBox, physical, etc.):**
AWS

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you think important (e.g. issue happens only occasionally):**

**Comment by lpinsivy** [comment]
Hi,

To send monitored data from pollers to the Centreon central server you can do this, using Centreon Broker like a proxy but to send configuration from Centreon central server to pollers you need a direct SSH access.

**Comment by DRAKUN** [comment]
yes i know, is it possible to send from node2 direct to central without direct access with this ???

**Comment by lpinsivy** [comment]
Hi,

You can do:

1. First method

Centreon --> SSH --> Poller2
Poller2 --> TCP (BBDO protocole) --> Centreon

2. Second method

Centreon --> SSH --> Poller2
Poller2 --> TCP (BBDO protocole) --> Poller1 --> TCP (BBDO protocole) --> Centreon

**Comment by DRAKUN** [comment]
so good i will try it and come back later if i've some issues

**Comment by DRAKUN** [comment]
Direct connection between CENTRAL and poller 2 is impossible. It's mandatory to passed in POLLER1

**Comment by lpinsivy** [comment]
@DRAKUN this is not regular behaviour but you can configure the Poller2 on Centreon Web to send configuration files in a special directory on Poller1 and on Poller1 to forward configuration of Poller2 on files change then restart centengine process on Poller2

Centreon --> SSH --> Poller2
Poller2 --> TCP (BBDO protocole) --> Centreon

Second method
Centreon --> SSH (Poller2 files)  --> Poller1 SSH (Poller2 files + centengine restart)  --> Poller2
Poller2 --> TCP (BBDO protocole) --> Poller1 --> TCP (BBDO protocole) --> Centreon
